source/community.applications/usr/local/emhttp/plugins/community.applications/Apps.page

```diff
@@ -145,22 +145,71 @@ function tr($string,$ret=false) {
     echo $string;
 }
 
-?>
-<style>
+$font = $display['font'] ?: "62.5";
+$themeOffset = ($display['theme'] == "azure" || $display['theme'] == "gray") ? $font + 10 : 0;
 
-<?
- include ("/$docroot/plugins/$plugin/skins/Narrow/css.php");
+switch ($font) {
+  case "50":
+    $doubleRow = 827;
+    $noMenu = 500;
+    break;
+  case "56.25":
+    $doubleRow = 922;
+    $noMenu = 558;
+    break;
+  case "62.5":
+    $doubleRow = 1016;
+    $noMenu = 608;
+    break;
+  case "68.75":
+    $doubleRow = 1111;
+    $noMenu = 653;
+    break;
+  case "75":
+    $doubleRow = 1205;
+    $noMenu = 719;
+    break;
+  case "80":
+    $doubleRow = 1281;
+    $noMenu = 763;
+    break;
+    
+}
+$doubleRow += $themeOffset;
+$noMenu += $themeOffset;
 ?>
-</style>
+
+
+
+<link rel='stylesheet' href='<?autov("/plugins/community.applications/skins/Narrow/themes/theme-$theme.css");?>'>
```

Comment:
Could potentially simplify `themes/theme-$theme.css` into `themes/$theme.css`

That would match the individual CSS var files I did for the webgui.

Reply:
Somewhat surprised that you didn't question /skins/Narrow/themes/theme-$theme.css

"Narrow" is a hold-over because at one point there were actually 3 different skins to choose from:

Table / list view (didn't last long)
Narrow (Basically icon only)
Wide (What we see now)

Jon insisted that Narrow was what everyone would want to see (I disagreed), so it was the default.  Since everyone hated it, I dropped it and went with strictly wide.  Naming it Narrow was simply because of how the default was on previous CA versions.

But I do still support 3rd party CSS overrides if someone ever chooses to take advantage of that (can't remember if Theme Engine does create the applicable file(s) for CA